src/templates/finops-hub/modules/dataFactory.bicep

```
@@ -4755,19 +4633,36 @@ resource pipeline_ToDataExplorer 'Microsoft.DataFactory/factories/pipelines@2018
 }
 
 //------------------------------------------------------------------------------
-// ingestion_RerunETL pipeline
-// Triggered manually
+// ingestion_ETL_dataExplorer pipeline
+// Queues ingestion_ETL_dataExplorer pipeline for each file in the container.
 //------------------------------------------------------------------------------
-@description('Safely reruns the ingestion_ETL_dataExplorer pipeline avoiding data duplication by.')
-resource pipeline_RerunETL 'Microsoft.DataFactory/factories/pipelines@2018-06-01' = if (deployDataExplorer) {
-  name: '${safeIngestionContainerName}_RerunETL'
+@description('Queues the ingestion_ETL_dataExplorer pipeline.')
```

nit: We should probably keep the context on why we're triggering this.
```suggestion
@description('Queues the ingestion_ETL_dataExplorer pipeline to account for Data Factory pipeline trigger limits..')
```